tools/lsan_blacklist.txt

```diff
@@ -0,0 +1,2 @@
+leak:__GI___strdup
+leak:__res_vinit
```

Comment:
Just curious, what's the purpose of blacklisting strdup()?  Apropos `__res_vinit()`, I've never seen that leak myself.  Are you seeing that with a particular version of glibc?